app/file_characterization_services/tika_file_characterization_service.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Class for Apache Tika based file characterization service
+# defines the Apache Tika based characterization service a ValkyrieFileCharacterization service
+# @since 0.1.0
+class TikaFileCharacterizationService
+  class Factory
+    def new(file_node, persister)
```

Comment:
This isn't actually doing anything useful (not helping conform to an interface). Can you get rid of it, and just call #new directly on TikaFileCharacteriztaionService?